docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -151,16 +151,16 @@ Will return a JSON object of the form:
 {
   "Images" : [
     {
-      "Candidate": "123456",
-      "PSCID": "MTL001",
-      "Visit": "V1",
-      "Visit_date": "2016-08-09", /* The date of the session. This will be null for phantoms and session that are not yet started */
```

Comment:
Why are all these values changed?

Reply:
I'm not sure - I took it directly from Xavier's original changes in his PR. I can revert them though if they aren't necessary.